elide-spring/elide-spring-boot-autoconfigure/src/test/java/example/tests/ControllerTest.java

```diff
@@ -221,26 +231,499 @@ public void jsonForbiddenApiPatchTest() {
+                                )
+                        )
+                )
+                .when()
+                .post("/json/operations")
+                .then()
+                .statusCode(HttpStatus.SC_OK)
+                .extract();
+        String result = deleteResponse.asString();
+        String expected = """
+                {"atomic:results":[{"data":null},{"data":null}]}""";
+        assertEquals(expected, result);
+
+    }
+
+    @Test
+    public void jsonApiAtomicOperationsExtensionPathInferTest() {
+        ExtractableResponse<Response> response = given()
+                .contentType(JsonApiController.JSON_API_ATOMIC_OPERATIONS_CONTENT_TYPE)
+                .accept(JsonApiController.JSON_API_ATOMIC_OPERATIONS_CONTENT_TYPE)
+                .body(
+                        atomicOperations(
+                                atomicOperation(AtomicOperationCode.add,
+                                        datum(resource(
+                                                type("group"),
+                                                id("com.example.operationsinfer1"),
+                                                attributes(
+                                                        attr("commonName", "Foo1")
+                                                )
+                                        ))
+                                ),
+                                atomicOperation(AtomicOperationCode.add,
+                                        datum(resource(
+                                                type("group"),
+                                                id("com.example.operationsinfer2"),
+                                                attributes(
+                                                        attr("commonName", "Foo2")
+                                                )
+                                        ))
+                                ),
+                                atomicOperation(AtomicOperationCode.update,
+                                        datum(resource(
+                                                type("group"),
+                                                id("com.example.operationsinfer2"),
+                                                attributes(
+                                                        attr("description", "Updated Description")
+                                                )
+                                        ))
+                                )
+                        )
+                )
+                .when()
+                .post("/json/operations")
+                .then()
+                .statusCode(HttpStatus.SC_OK)
+                .extract();
+        Map<String, Object> attributes = response.path("'atomic:results'[1].data.attributes");
+        assertEquals("Foo2", attributes.get("commonName"));
+        assertEquals("Updated Description", attributes.get("description"));
+
```

Comment:
Same comment.  Do another round trip request and verify the response matches what we expect (to confirm the database was updated correctly).

Reply:
I have updated the test cases.